internal/communitymodules/merge_test.go

```diff
@@ -0,0 +1,60 @@
+package communitymodules
+
+import (
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestMergeRowMaps(t *testing.T) {
```

Comment:
I can't see tests for other cases i.e:
- more than one key inside `moduleMap`,
- merging maps with some different keys.

Now such data are probably impossible but we don't know about future. Tested module is generic library that should support such cases.

Reply:
Added case covering those